src/proxy/internal/Makefile

```diff
@@ -0,0 +1,12 @@
+export PROTOC?=protoc
+export RM?=rm
+
+.PHONY: build clean proto
+
+clean:
+	$(RM) -rf *.pb.go
+
+proto:
+	$(PROTOC) --go_out=plugins=grpc,import_path=internal:. *.proto
```

Comment:
What do you think about using of `go generate`?

Reply:
go generate bring no advantages over make and have obvious disadvantages of
- not having single default file for build rules specification, similar to Makefile;
- not performing globbing;
- be designed to execute the commands serial only;
- not be able to track dependency between build commands.